connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -0,0 +1,217 @@
+openapi: 3.0.0
+info:
+  description: |
+    Connector Builder Server API
+
+  version: "1.0.0"
+  title: Connector Builder Server API
+  contact:
+    email: contact@airbyte.io
+  license:
+    name: MIT
+    url: "https://opensource.org/licenses/MIT"
+externalDocs:
+  description: Find out more about Connector Builder
+  url: "https://docs.airbyte.com/connector-development/config-based/overview/"
+
+paths:
+  /v1/stream/read:
+    post:
+      summary: Reads a specific stream in the source
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/StreamReadRequestBody"
+        required: true
+      responses:
+        "200":
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/StreamRead"
+        "400":
+          $ref: "#/components/responses/ExceptionResponse"
+        "422":
+          $ref: "#/components/responses/InvalidInputResponse"
+  /v1/streams/list:
+    post:
+      summary: List all streams present in the connector definition, along with their specific request URLs
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/StreamsListRequestBody"
+        required: true
+      responses:
+        "200":
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/StreamsListRead"
+        "400":
+          $ref: "#/components/responses/ExceptionResponse"
+        "422":
+          $ref: "#/components/responses/InvalidInputResponse"
+
+components:
+  schemas:
+    StreamRead:
+      type: object
+      required:
+        - results
+      properties:
+        results:
+          type: array
+          description: The RECORD and STATE AirbyteMessages coming from the read operation
+          items:
+            type: string
+        logs:
+          type: array
+          description: The LOG AirbyteMessages coming from the read operation
+          items:
+            type: string
```

Comment:
I set these to `string`, because I didn't seen an obvious way to reference just the AirbyteMessage or AirbyteLogMessage schema from [the protocol](https://github.com/airbytehq/airbyte/blob/master/airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml) file, but I'm open to suggestions here

Reply:
I figured out how to do this after all! Just needed to reference the whole airbyte protocol in one schema, and then add more slashes when referencing that schema to access its children